netneurotools/metrics.py

```diff
@@ -99,3 +101,87 @@ def communicability_wei(adjacency):
     cmc[np.diag_indices_from(cmc)] = 0
 
     return cmc
+
+
+def rich_feeder_peripheral(x, sc, stat='median'):
+    """
+    Calculates median or mean "connectivity" in rich (hub to hub), feeder
+    (hub to non-hub), and peripheral (non-hub to non-hub) links.
+
+    This function takes a square symmetric correlation/connectivity matrix `x`
+    and computes the median or mean value within rich, feeder, and peripheral
+    links. Hubs are defined againt a threshold (from 0 to the maximum degree
+    `k`, defined on the structural connectivity matrix, `sc`).
+
+    Parameters
+    ----------
+    x : (N, N) numpy.ndarray
+        Symmetric correlation or connectivity matrix
+    sc : (N, N) numpy.ndarray
+        binary structural connectivity matrix
```

Comment:
```suggestion
        Binary structural connectivity matrix
```

(Capitalization just for consistency sake)